angular/core/models/membership_model.coffee

```diff
@@ -4,7 +4,7 @@ angular.module('loomioApp').factory 'MembershipModel', (BaseModel, AppConfig) ->
     @plural: 'memberships'
     @indices: ['id', 'userId', 'groupId']
     @searchableFields: ['userName', 'userUsername']
-    @serializableAttributes: AppConfig.permittedParams.membership
```

Comment:
volume on discussion is a function (which returns discussion_reader_volume or membership_volume), volume on membership is a value. To give them the same interface we need volume on membership to be a function, so we moved the value to volume_value.